java8/start-jar

```diff
@@ -8,5 +8,5 @@ XMX=${JAVA_MAX_MEMORY-128}
 MAXPERM=${JAVA_MAX_PERM-128}
 
 cd /home/application/current
-
-mvn clean package jetty:run
+java -XX:MaxPermSize=${MAXPERM}m -Xmx${XMX}m -jar tsuru-app.jar -Dserver.host=0.0.0.0 -Dserver.port=${PORT}
```

Comment:
Also added a gradle task. This only works for gradle projects. With this it is possible to use git push also. The only thing you have to do is add a stage tasks to your build.gradle. For ratpack the following example:

```groovy
task stage() {
  dependsOn shadowJar
}

tasks.stage.doLast() {
  def name = "build/libs/${project.name}-${shadowJar.classifier}.jar"
  copy {
    from(name) {
      rename { 'current.jar' } 
    }
    into '.'
  }
}
```

See also https://devcenter.heroku.com/articles/deploying-gradle-apps-on-heroku perhaps a gradle platform should be created.